test/test_parmed_openmm.py

```diff
@@ -93,6 +93,22 @@ def test_load_topology(self):
         self.assertEqual(len(parm.residues), len(structure.residues))
         self.assertEqual(len(parm.bonds), len(structure.bonds))
 
+    def test_load_topology_uncondensed_atom_types(self):
+        """ Tests condense_atom_types arg """
+        ommparm = app.AmberPrmtopFile(get_fn('complex.prmtop'))
+        parm = load_file(get_fn('complex.prmtop'))
+        system = ommparm.createSystem(implicitSolvent=app.OBC1)
+        structure_condensed = openmm.load_topology(ommparm.topology, system, condense_atom_types=True)
+        structure_uncondensed = openmm.load_topology(ommparm.topology, system, condense_atom_types=False)
+
+        num_unique_atom_types_condensed = len(set([a.atom_type for a in structure_condensed]))
+        num_unique_atom_types_uncondensed = len(set([a.atom_type for a in structure_uncondensed]))
+
+        assert num_unique_atom_types_uncondensed > num_unique_atom_types_condensed
+
+        # This may change if this or a similar flag does a partial condensing, see PR #1087
+        assert num_unique_atom_types_uncondensed == len([*structure_uncondensed.atoms])
```

Comment:
This is causing a SyntaxError in the pypy test:

```
Traceback (most recent call last):
  File "/home/travis/.pyenv/versions/pypy2.7-6.0.0/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/.pyenv/versions/pypy2.7-6.0.0/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/.pyenv/versions/pypy2.7-6.0.0/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/ParmEd/ParmEd/test/test_parmed_openmm.py", line 110
    assert num_unique_atom_types_uncondensed == len([*structure_uncondensed.atoms])
```

Why not just use `len(structure_uncondensed.atoms)`?  It's a `list`-derived type.